src/ErrorProne.NET.StructAnalyzers.Tests/HiddenStructCopyAnalyzerTests.cs

```diff
@@ -75,6 +75,21 @@ public static int GetValue(in TestStruct test)
     }";
             await VerifyCS.VerifyAsync(code);
         }
+        
+        [Test]
+        public async Task WarnForOperators()
+        {
+            string code = @"
+public struct S
+    {
+        private long l1, l2, l3;
+
+        public int GetValue() => 42;
+        public static int operator+(in S lhs, in S rhs) => lhs.[|GetValue|]() + rhs.[|GetValue|]();
```

Comment:
❔ Are you sure this is the test you intended? I was expecting to see an invocation of `S.operator+` producing a warning.

Reply:
e.g.:

```
struct S {
  private long a, b, c;

  public static int operator+(S left, S right) => throw null;

  static int Method(in S left, in S right)
    => left + right; // Warning?
}
```

Reply:
I actually not 100% sure about that.

I'm not sure whether to treat `left + right` as a hidden copy or not. Technically, this is just a form of method invocations, and we don't want to emit diagnostics for `left.Plus(right)` when the `Plus` method takes a struct by value.

We do have an analyzer to suggest that its more efficient to pass the struct by `in`, but I'm not sure we should emit warning about the hidden copy.

BTW, the operators were missing from `ESP05` rule, i.e. the following code had no issues:
```csharp
public static int operator+(S left, S right) => throw null;
```
But after my latest commit, the diagnostics are emitted to suggest passing `left` and `right` by `in`.